src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -354,3 +354,133 @@ UicOperatingPeriod/ValidDayBits | | | see (1) below
 This is a string sequence of `0` (inactive days) and `1` (active days).  The
 sequence should contains as many `0/1` as there is days in the period.  The
 first character correspond to the first day of the period.
+
+## ligne_<line_id_md5>.xml
+
+The name of the file is generated after the `line_id` by computing a MD5
+checksum of the `line_id`. For example, for a line whose ID is `bob`, the file
+would be named `line_e01096b9ffe3f416157f6ec46c467725.xml`.
+
+### Top level structure
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame 
+		id="FR:GeneralFrame:<line_id>:<stop_provider_code>" 
+		version="any">
+	<members>
+		<!-- One Route for each 'route_id' -->
+		<Route />
+		<ServiceJourneyPattern />
+		<!-- One ScheduledStopPoint for each Stop in a Journey Pattern -->
+		<ScheduledStopPoint />
+		<!-- One ScheduledStopPoint for each Stop in a Journey Pattern -->
+		<PassengerStopAssignment />
+		<!-- One ServiceJourney for each 'trip_id' -->
+		<ServiceJourney />
+	</members>
+</GeneralFrame>
+```
+
+### Route
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+Route/@id | routes.txt | route_id | see [id formatting](#id-of-objects)
+Route/@version | | | fixed value `any`
+Route/Name | routes.txt | route_name |
+Route/Distance | | | fixed value `0`
+Route/LineRef/@ref | routes.txt | line_id | see [id formatting](#id-of-objects) with `line` as object type
+Route/DirectionType | routes.txt | direction_type | see (1) below
+
+**(1) Direction Type**
+
+Use the following mapping to convert the direction type from NTFS to NeTEx
+France.
+
+NTFS Direction Type | NeTEx Direction Type
+--- | ---
+forward | inbound
+backward | outbound
+inbound | inbound
+outbound | outbound
+clockwise | clockwise
+anticlockwise | anticlockwise
+
+### ServiceJourneyPattern
+
+A Journey Pattern is created for each unique sequence of stops in the Vehicle
+Journeys.
```

Comment:
Maybe adding pickup/dropoff or ITL issues to this definition ?

Reply:
Done in fdad31a95099e20bbb98237024dee27fbb3c61ec. Can you check that these are the only fields concerned? I'm not sure about some of the others (for example, `alighting_duration` and `boarding_duration`)?

Reply:
IMHO it's ok like that